core-bundle/src/Resources/config/services.yml

```diff
@@ -1119,7 +1120,12 @@ services:
             version: 4.13
             message: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "contao.csrf.token_manager" instead.
 
-    Contao\CoreBundle\Framework\ContaoFrameworkInterface: '@contao.framework'
+    Contao\CoreBundle\Framework\ContaoFrameworkInterface:
+        alias: contao.framework
+        deprecated:
+            package: contao/core-bundle
+            version: 4.7
+            message: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "contao.framework" instead.
```

Comment:
This will need to be backported to Contao 4.9 as well.

Reply:
see https://github.com/contao/contao/pull/4199